client/src/partials/accounts/create_account/create.js

```diff
@@ -96,6 +96,14 @@ angular.module('bhima.controllers')
         account.is_asset = null;
       }
 
+      if(account.is_charge === 'true'){
+        account.is_charge = 1;
+      } else if (account.is_charge === 'false'){
+        account.is_charge= 0;
+      } else {
+        account.is_charge = null;
+      }
```

Comment:
You can condense this into one line! 
```js
account.is_charge = (account.is_charge === 'true') ? 1 : (account.is_charge === 'false') ? 0 : null;
```

However, I recommend that we do not use the strings 'true' and 'false'.  If you are using a checkbox, you can set `ng-true-value=1` and `ng-false-values=0` on the page where these are defined.  If they are coming from the server, they should be stored as a TINYINT field in the database.